less/mixins.less

```diff
@@ -1 +1,29 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.link-bland() {
+    text-decoration: none;
+    color: @river-bend;
+}
+
+.desktop-width(){
+    max-width: 800px;
+  width: 100%;
+  margin: 0 auto;  
+}
+
+
```

Comment:
Generally speaking. I like to give me all of my parameters default values in case I forgot to put one in when I call it. It doesn't break less.